src/traces/histogram/calc.js

```diff
@@ -112,6 +113,8 @@ module.exports = function calc(gd, trace) {
     // average and/or normalize the data, if needed
     if(doavg) total = doAvg(size, counts);
     if(normfunc) normfunc(size, total, inc);
+    if(trace.mode === 'cumulative') cdf(size);
```

Comment:
maybe `'cumulative'` should have a different meaning for `histnorm !== ''` and `histfunc !== 'count'` ?